src/core/pad.h

```diff
@@ -68,6 +69,13 @@ class Pads {
 
     void setLua(Lua L);
 
+    bool isPadConnected(int pad) {
+        if (pad > m_pads.size())
```

Comment:
Please fully decorate with {}

Reply:
Do you decorate only if-else or any if statement in general? Particularly, something like `if (condition) return;`

Reply:
Basically, if it goes on two lines for any reason, then braces should appear. If it's single-line and simple enough, then it's okay not to have braces.

Multi-lines no-braces is a pitfall: https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/

Reply:
Cool, just seeing where you stand on that. I always try to be pretty explicit with brackets, namespaces, etc., goto fail being a big inspiration - was just pondering how broad to interpret this(and how much code to re-decorate).